plugins/rapid7_insightidr/komand_rapid7_insightidr/util/formatting.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+
+def refactor_message(message: dict):
+    new_message = json.loads(str(message).replace("\n", "\\n").replace("'", '"'))
+    return new_message
```

Comment:
Add your return type hint and add a doc string for this function
```suggestion
def refactor_message(message: dict) -> return type:
    """
    <x,y,z>
    :param <>: ...
    :return: ...
    """
    new_message = json.loads(str(message).replace("\n", "\\n").replace("'", '"'))
    return new_message
```